src/shogun/machine/BaggingMachine.cpp

```diff
@@ -177,7 +178,8 @@ bool CBaggingMachine::train_machine(CFeatures* data)
 	for (index_t i = 0; i < m_num_bags*m_bag_size; ++i)
 		rnd_indicies.matrix[i] = CMath::random(0, m_bag_size-1);
 
-	#pragma omp parallel for
+	auto pb = SG_PROGRESS(range(m_num_bags));
+#pragma omp parallel for
 	for (int32_t i = 0; i < m_num_bags; ++i)
```

Comment:
if omp is used, I assume the `pb` definition needs to happen before (and cannot be done in the for loop?) @geektoni ?

Reply:
Yes, the ``pb`` definition has to appear before the ``#pragma``.